jest.config.ts

```diff
@@ -0,0 +1,19 @@
+import type { Config } from "jest";
+
+const config: Config = {
+  testEnvironment: "jest-environment-jsdom",
+  transform: {
+    "^.+\\.tsx?$": "ts-jest",
+  },
+  moduleNameMapper: {
+    '\\.scss$': 'identity-obj-proxy'
+  },
+  setupFilesAfterEnv: ["<rootDir>/src/setupTests.ts"],
+  testRegex: "(/__tests__/.*|(\\.|/)(test|spec))\\.tsx?$",
```

Comment:
It looks like jest is trying to run cypress tests as well, please find the way to exclude it. For example, you could use `test` for jest and `spec` for Cypress or vice versa.